tests/snappi_tests/multidut/pfc/files/multidut_helper.py

```diff
@@ -77,58 +87,126 @@ def run_pfc_test(api,
     stop_pfcwd(duthost2, tx_port['asic_value'])
     disable_packet_aging(duthost2)
 
-    """ Rate percent must be an integer """
-    test_flow_rate_percent = int(TEST_FLOW_AGGR_RATE_PERCENT / len(test_prio_list))
+    global DATA_FLOW_DURATION_SEC
+    global data_flow_delay_sec
+
+    dut_port = snappi_extra_params.multi_dut_params.multi_dut_ports[0]['peer_port']
+
+    port_id = 0
```

Comment:
Can you add a comment to indiucate why port id here is 0

Reply:
@developfast the port _id is mentioned 0 here because the in the setup_base_traffic_config() in traffic_generation.py, i am pointing it to use the first port as rx_port, so rx_port_id is 0 and tx_port_id is 1